src/components/EventList.tsx

```diff
@@ -53,9 +59,16 @@ const EventList = ({ events, org } : EventListProps) : JSX.Element => {
                             />
                         </View>
                         <View data-test="location-title">{ e.location.title }</View>
-                        <Button data-test="sign-up-button" variant="cta">
+                        <Button data-test="sign-up-button" marginTop="size-50" variant="cta">
                             <Msg id="misc.eventList.signup"/>
                         </Button>
+                        <NextLink href={ `/o/${org?.id}/events/${e.id}` }>
+                            <a>
```

Comment:
Yes, I think we've discussed this previously. `next/link` is structured this way, but I've seen examples wrapping the `Link` (`NextLink` in our case) in a custom component which makes links less verbose.

Reply:
Thinking about the resulting HTML, this would (most likely?) result in a `<button>` inside an `<a>` which feels weird to me. But I guess this is only an issue if we're using buttons, and you're right (in another comment) that we don't need to figure that out right now.